src/SignalR/docs/specs/TransportProtocols.md

```diff
@@ -186,6 +186,32 @@ Long Polling requires that the client poll the server for new messages. Unlike t
 
 A Poll is established by sending an HTTP GET request to `[endpoint-base]` with the following query string parameters
 
+## HTTP Streaming (HTTP/1, HTTP/2, HTTP/3)
+
+HTTP Streaming uses a single bi-directional HTTP request or 2 uni-directional HTTP requests as a single connection (one for input, one for output). If the HTTP client in use supports bi-directional streaming, negotiation can be skipped (if the client knows the server supports the streaming transport), otherwise, it requires a connection already be established using the `POST [endpoint-base]/negotiate` request.
+
+### 1 bi-directional HTTP Request
+
+For clients that support bi-directional streaming, the HTTP Streaming transport is activated by making a POST request `[endpoint-base]` with the content type `application/bedrock-streaming`. The **optional** `id` query string value is used to identify the connection to attach to. 
```

Comment:
`application/bedrock-streaming` Hooooo boy that name ;).

Reply:
Yea that name sucks